scripts/configure

```diff
@@ -23,14 +24,11 @@ function load_dev_data() {
     docker-compose \
         exec -T django \
         wget -q \
-            -O /data/VA_data.zip \
+            -O /data/districtbuilder_data.zip \
             https://s3.amazonaws.com/districtbuilderdata/VA_data.zip
```

Comment:
I think we might want to parametrize the `DISTRICTBUILDER_DATA_URL` as well. Additionally, did we agree to assume that the data would be stored on S3? If so, I'd rather use `aws s3 cp` over `wget` so that the data bucket doesn't have to be publicly accessible.

Reply:
My recollection is that we ended up agreeing that the data would be placed in a designated local directory, which is why I opted not to parametrize this, because I think this script is unique to development setup. I think the same thing would apply to the `wget` vs. `s3` choice, but if I'm misremembering, let me know.

Reply:
I don't think you're misremembering, that sounds right to me.